games-emulation/pcsx_rearmed_libretro/pcsx_rearmed_libretro-r22_20180426.recipe

```diff
@@ -0,0 +1,57 @@
+SUMMARY="A port of PCSX ReArmed, a PlayStation emulator to the libretro API"
+DESCRIPTION="Mupen64Plus is a cross-platform Sony PlayStation emulator which \
```

Comment:
...

Reply:
Reminder: `../mupen64plus_libretro/mupen64plus_libretro-2.5_20171119.recipe` has this description:
```
DESCRIPTION="Mupen64Plus is a cross-platform plugin-based N64 emulator which \
is capable of accurately playing many games. This is the libretro core."
```

My guess is that the following tiny change might be sufficient:
```diff
-DESCRIPTION="Mupen64Plus is a cross-platform Sony PlayStation emulator which \
+DESCRIPTION="PCSX ReARMed is a cross-platform Sony PlayStation emulator which \
 is capable of accurately playing many games. This is the libretro core."
```
So, if you wish I can push this `s/Mupen64Plus/PCSX ReARMed/` here, unless you would like to make other changes in the description, of course.
(There's no urgency. I'm just suggesting this in case the above change would be ok.)